packages/components/src/vue/components/word-list/download-confirmation.vue

```diff
@@ -175,8 +175,8 @@
           dataForDownload = this.prepareDownloadListFull()
         }
 
-        const result = Download.collectionToCSV(dataForDownload.delimiter, dataForDownload.exportFields)(dataForDownload.wordlistData)
-        Download.downloadBlob(result, `wordlist-${this.languageCode}.csv`)
+        const result = Download.collectionToCSV(dataForDownload.delimiter, dataForDownload.exportFields, false)(dataForDownload.wordlistData)
+        Download.downloadBlob(result, `wordlist-${this.languageCode}.tsv`)

```

Comment:
Hmm. This changes it for both downloads, right? The tsv extension only really applies for the flashcard download though.  It should stay csv for the other.

Reply:
you are quite right - I missed it! thank you for the catch
fixed